sdk/core/Azure.Core/src/Diagnostics/AzureCoreEventSource.cs

```diff
@@ -78,6 +79,7 @@ public void RequestContent(string requestId, byte[] content, Encoding? textEncod
         }
 
         [Event(RequestContentEvent, Level = EventLevel.Verbose, Message = "Request [{0}] content: {1}")]
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026", Justification = "WriteEvent is used with an array with primitive type elements.")]
         public void RequestContent(string requestId, byte[] content)
```

Comment:
byte[] is not a primitive type but it seems that arrays of primitive types are also acceptable based on: https://github.com/dotnet/runtime/blob/ab2b80d06dd4d997df5ffa72a3c4a99cb36ccbff/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/Statics.cs#L421